docs/UserGuide.md

```diff
@@ -47,44 +53,86 @@ NetworkBook 1.3 includes several new features and improvements for ease of use.
 
 1. Make sure you have Java 11 installed on your computer.
     * You can check that it is installed by running the command `java --version` in the command prompt/terminal.
-1. Download the latest version of NetworkBook (`networkbook.jar`) from the top of the [NetworkBook GitHub releases page](https://github.com/AY2324S1-CS2103T-T08-2/tp/releases).
-1. Create a new folder on your computer (e.g. on your computer desktop). You can name this folder however you want.
-1. Move the `networkbook.jar` file into the folder you just created.
-1. Open NetworkBook:
+2. Download the latest version of NetworkBook (`networkbook.jar`) from the top of the [NetworkBook GitHub releases page](https://github.com/AY2324S1-CS2103T-T08-2/tp/releases).
+3. Create a new folder on your computer (e.g. on your computer desktop). You can name this folder however you want.
+4. Move the `networkbook.jar` file into the folder you just created.
+5. Open NetworkBook:
```

Comment:
Markdown language automatically converts the numbering into the correct numbers.
It is a better practice to leave the numbering all at `1` so that in case anyone wants to insert a step in between, he does not have to change all the numbering.
Please revert all numberings back to `1`.

Reply:
I see, will revert the numberings.